### PR TITLE
ARTEMIS-2541 Improve message browser of Admin UI

### PR DESCRIPTION
**Pull request by sebthom**

### CODE REVIEW

**Comment by clebertsuconic** [comment]
any chance you could attach before and after images to the PR discussion?

**Comment by sebthom** [comment]
@clebertsuconic I added another screenshot before the modifications.

File: artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js [hunk 1]
```diff
@@ -26,44 +26,55 @@ var ARTEMIS = (function(ARTEMIS) {
...
+         width: '50px'
+      }, {
+         field: 'persistentSize',
+         displayName: 'Size',
+         cellTemplate: '<div class="ngCellText" title="{{row.entity.persistentSize}}">{{formatPersistentSize(row.entity.persistentSize)}}</div>',
+         width: '100px'
+      }, {
+         field: 'userID',
+         displayName: 'User ID',
+         cellTemplate: '<div class="ngCellText">{{row.entity.userID ? row.entity.userID : row.entity.StringProperties._AMQ_VALIDATED_USER}}</div>',
```
Comment (sebthom) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js]: @clebertsuconic I actually would prefer to show the _AMQ_VALIDATED_USER value by default and falling back to userID if it is not present. WDYT?

Comment (clebertsuconic) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js]: thanks.. I didnt realize that.. thakns

Comment (clebertsuconic) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js]: > @clebertsuconic I added another screenshot before the modifications to the JIRA issue.
thanks a lot, I didn't realize that before.

Comment (clebertsuconic) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js]: > @clebertsuconic I actually would prefer to show the _AMQ_VALIDATED_USER value by default and falling back to userID if it is not present. WDYT?

would that be intuitive to users? shouldn't we keep it separated columns?
as long as users can understand the result without having to read any doc about it.

Comment (sebthom) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js]: > > @clebertsuconic I actually would prefer to show the _AMQ_VALIDATED_USER value by default and falling back to userID if it is not present. WDYT?
> 
> would that be intuitive to users? shouldn't we keep it separated columns?
> as long as users can understand the result without having to read any doc about it.

Sure, we could do that too.

Comment (sebthom) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js]: @clebertsuconic I added a separated validated user column and updated the screenshot in the JIRA issue.

**Comment by clebertsuconic** [comment]
I checked AMQP, and this property is only filled with Core Protocol. I checked and it worked fine.

Perhaps we could add with extra properties.

**Comment by clebertsuconic** [comment]
but that will be an improvement. I'm merging this.

**Pull request closed without merging**